### ISSUE TITLE
Error with the new IP pricing

### ISSUE THREAD

**Issue by aaronriedel** [jangraefen/hcloud-pricing-exporter#39]
Hey there, I just updated the container to the newest version and unfortunately it crashes with the following error:
```
goroutine 1 [running]:
github.com/jangraefen/hcloud-pricing-exporter/fetcher.Fetchers.MustRun(...)
	/app/fetcher/fetcher.go:97
main.main()
	/app/main.go:62 +0xbae
panic: 1 error(s) occurred:
* no primary IP pricing found for datacenter fsn1-dc14
```

Maybe this only affects instances in fsn1-dc14?

(btw thank you so much for this tool, it worked great for me in the last months)

**Comment by jangraefen** [comment]
Hey @aaronriedel,

Glad it worked great for you so far and thanks for letting me know. Since nobody else is reporting this issue and I have not found it in testing, my guess is that not all data centers are affected, as you assumed.

I am currently away from home for the holidays, but will have a look as soon as I get back in a few days. If you have time and want a fix sooner, I am also more then happy to review a PR for this bug.

Thanks again for the report and happy holidays 🙂.

**Comment by aaronriedel** [comment]
Thank you so much for the quick response! Enjoy your holidays and I will see if I can debug it a bit further in the coming days.

**Comment by jangraefen** [comment]
Hey @aaronriedel,

thank you again for your report and thanks to @byReqz contributing the fix. I just published v0.6.0 and the matching helm chart with the needed fix. Maybe you could quickly confirm that it works now?

I also added end-to-end tests to ensure a higher quality standard in the future. Sorry again for the inconviences...

**Comment by aaronriedel** [comment]
Thank you so much @jangraefen 
I was stuck in debugging and asked @byReqz for help, apparently he was the right guy to ask :D I am happy that it works again.
